.circleci/config.yml

```diff
@@ -17,33 +17,26 @@ jobs:
       - run:
           name: Run test for Python << parameters.version >>
           command: |
+            # Install surround
             sudo python setup.py install
+
+            # Run tests
             sudo python setup.py test
 
+            # Install requirements
             sudo pip3 install pylint==2.3.0
             sudo pip3 install Flask==1.0.2
             sudo pip3 install gunicorn==19.9.0
             sudo pip3 install tornado==6.0.1
 
+            # Run pylint tests
             pylint setup.py
-
             find surround/ -iname "*.py" | xargs pylint
             find examples/ -iname "*.py" | xargs pylint
 
+            # Run examples
             ls examples/ | xargs -n 1 -I '{}' python examples/'{}'/main.py
 
-            mkdir ./temp
-            surround tutorial ./temp
-            surround init ./temp -p project -d "A test project"
```

Comment:
This creates a non-tutorial project. Is there any reason why this is removed in this PR?

Reply:
Yes, this test is present in https://github.com/a2i2/surround/blob/8d726fd850811b3fac0e473a88dbdfa0bab3efd2/surround/tests/cli/project_cli/init_test.py#L11